src/openrct2/scripting/bindings/world/ScPark.cpp

```diff
@@ -401,6 +401,21 @@ namespace OpenRCT2::Scripting
         }
     }
 
+    std::vector<int32_t> ScPark::getMonthlyExpenditure(const std::string& expenditureType) const
+    {
+        std::vector<int32_t> result(kExpenditureTableMonthCount, 0);
```

Comment:
Documentation suggests this vector will be up to 16 entries long, but here you always create is 16-long and only fill in partially, suggesting some months may have passed with no expenditures. Please align both. I'd prefer to return properly sized vector here.